wandb/sdk/launch/utils.py

```diff
@@ -51,8 +51,8 @@ def _is_wandb_local_uri(uri: str) -> bool:
     return _WANDB_LOCAL_DEV_URI_REGEX.match(uri) is not None
 
 
-def _is_github_uri(uri: str) -> bool:
-    return _GIT_URI_REGEX.match(uri) is not None
+def _is_git_uri(uri: str) -> bool:
+    return bool(_GIT_URI_REGEX.match(uri))
```

Comment:
what was this change for? just wondering since the wandb regex check is still `is not None`

Reply:
This is cleaner, forgot to change the other ones, so I'll change those now